web/api/v0/__init__.py

```diff
@@ -940,3 +941,50 @@ def patch(self, token: str, password: str) -> ResponseReturnValue:
 
 
 api.add_resource(ResetPasswordResource, '/user/reset-password')
+
+
+class UserActiveResource(Resource):
+    @use_kwargs(
+        {
+            "name": fields.Str(required=True),
+            "fname": fields.Str(required=False, missing=None),
+            "byear": fields.Int(required=False, missing=None),
+            "uid": fields.Int(required=True),
+        },
+        location="form",
+    )
+    def post(self, name: str, fname: str, byear: int, uid: int) -> ResponseReturnValue:
```

Comment:
The type hints dont match the webargs annotation, the non-required ones should be | None